nautobot/core/utils/lookup.py

```diff
@@ -192,3 +192,31 @@ def get_table_for_model(model):
         (Union[Table, None]): Either the `Table` class or `None`
     """
     return get_related_class_for_model(model, module_name="tables", object_suffix="Table")
+
+
+def get_created_and_last_updated_usernames_for_model(instance):
+    """
+    Args:
+        instance: A model class instance
+
+    Returns:
+        created_by: Username of the user that created the instance
+        last_updated_by: Username of the user that last modified the instance
+    """
```

Comment:
😅

Reply:
TBH, I dont think you need a doctoring for this, looks quite self-explanatory to me, but if you choose I think you should have a description